app/(sign)/sign/[id]/components/SignSingleDocument.tsx

```
@@ -872,15 +884,22 @@ export default function SignSingleDocument({
+          <TextInputModal
+            isOpen={isModalOpen}
+            onClose={() => setIsModalOpen(false)}
+            onComplete={handleSignatureComplete}
+          />
+        ) : (
+          <SignatureModal
+            isOpen={isModalOpen}
+            onClose={() => setIsModalOpen(false)}
+            onComplete={handleSignatureComplete}
+            existingSignature={
+              localSignatures.find((s) => s.area_index === selectedArea)?.signature_data!
```

Biome (2.4.4)

[error] Forbidden non-null assertion after optional chaining.

(lint/suspicious/noNonNullAssertedOptionalChain)